docker/docker-compose.yml

```diff
@@ -99,30 +103,28 @@ services:
             - inventree_data:/home/inventree/data:z
         restart: unless-stopped
 
-    # nginx acts as a reverse proxy
-    # static files are served directly by nginx
-    # media files are served by nginx, although authentication is redirected to inventree-server
-    # web requests are redirected to gunicorn
-    # NOTE: You will need to provide a working nginx.conf file!
+    # caddy acts as reverse proxy and static file server
+    # https://hub.docker.com/_/caddy
     inventree-proxy:
-        image: nginx:stable
+        container_name: inventree-proxy
+        image: caddy:alpine
+        restart: always
         depends_on:
             - inventree-server
+        ports:
+            - 80:80
+            - 443:443
```

Comment:
I think it would be beneficial to be able to set the web port via environment file in case another server is used to proxy for inventree

Reply:
Done